workflows/epigenetics/cutandrun/cutandrun.ga

```diff
@@ -676,7 +671,7 @@
             },
             "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1",
             "tool_shed_repository": {
-                "changeset_revision": "ddf54b12c295",
+                "changeset_revision": "d698c222f354",
```

Comment:
This is indeed the latest version, and you'd get this version when installing the workflow, so it's probably not worth creating a new version if there's no actual change.

Reply:
I agree. However, I think we need to keep this PR open so autoupdate does not create a new one next time.

Reply:
Hmm, the autoupdate bot needs some love for this sort of things. Other bots allow you to close a PR under some conditions and will not re-open a new PR if there are no further changes. Or maybe let's try and see if it really opens a new PR with no new changes ?

Reply:
OK